python/src/opendp/context.py

```diff
@@ -300,29 +304,46 @@ def unit_of(
     >>> dp.unit_of(l1=2.0)
     (L1Distance(f64), 2.0)
 
-    :param contributions: Greatest number of records a privacy unit may contribute to microdata
+    :param contributions: Greatest number of records or identifiers a privacy unit may contribute to microdata
     :param changes: Greatest number of records a privacy unit may change in microdata
     :param absolute: Greatest absolute distance a privacy unit can influence a scalar aggregate data set
     :param l1: Greatest l1 distance a privacy unit can influence a vector aggregate data set
     :param l2: Greatest l2 distance a privacy unit can influence a vector aggregate data set
+    :param identifier: Name of the column that contains identifiers
```

Comment:
I was thinking the identifier could be composed of multiple columns? From this description it seems like it's just one.